config/pickit.ini

```diff
@@ -207,15 +206,15 @@ magic_ogre_gauntlets=0
 ;==================================================================;
 
 ;rare jewelry
-rare_amulet=1
+rare_amulet=1, (2_amazon_skills, 2_all_skills, 2_assasin_skills, 2_barbarian_skills, 2_Druid_skills, 2_necromancer_skills, 2_paladin_skills, 2_sorceress_skills)
 rare_jewel=1
-rare_ring=1
+rare_ring=1, (and(life_stolen, mana_stolen), faster_cast_rate)
 
 ;rare circlets
-rare_circlet=1
-rare_coronet=1
-rare_diadem=1
-rare_tiara=1
+rare_circlet=1, ((faster_cast_rate, 2_amazon_skills), (faster_cast_rate, 2_assasin_skills), (faster_cast_rate, 2_barbarian_skills), (faster_cast_rate, 2_Druid_skills), (faster_cast_rate, 2_necromancer_skills), (faster_cast_rate, 2_paladin_skills), (faster_cast_rate, 2_sorceress_skills), (2_all_skills, faster_cast_rate), (3_pcombat, faster_cast_rate), (faster_run_walk, 2_amazon_skills), (faster_run_walk, 2_all_skills), (faster_run_walk, 2_assasin_skills), (faster_run_walk, 2_barbarian_skills), (faster_run_walk, 2_Druid_skills)), ethereal
```

Comment:
Can rare circlets get 3_pcombat? I thought only magic can.

Reply:
The cant. Removed that part